### PR TITLE
update release instructions

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
I tried to cut through most of the noise, and updated for the new "don't
copy release notes three times" idea.

CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: release/RELEASE.md [hunk 1]
```diff
@@ -121,24 +55,20 @@ patches we backport to the 1.0 release branch).
...
-      - The above link takes you docs.daml.com's "getting started" installation guide;
-    - [Maven](https://maven.apache.org/install.html)
-      - You may have to manually set the  environment variable `JAVA_HOME`;
-      - For example, assuming the Zulu Java-SDK, something like
-        `C:\Program Files\Zulu\zulu-14`;
     - [Node.js](https://nodejs.org/en/download/)
-      - Just the bare install; you don't need Visual Studio build
-        tools for compiling C dependencies (and trying to install them
-        takes forever and in the end hangs it seems);
+      - Just the bare install; no need to build C dependencies.
```
Comment (cocreature) [inline release/RELEASE.md:61]: Does the yarn install not bundle nodejs on Windows?

**Comment by cocreature** [comment]
I believe we also need to update our release checks to no longer require modifications to the release notes file.

**Comment by cocreature** [comment]
And I think the docs cronjob also needs updating.

**Comment by garyverhaegen-da** [comment]
> I believe we also need to update our release checks to no longer require modifications to the release notes file.
> And I think the docs cronjob also needs updating.

We don't actually need to update either. For the release checks, changing the release-notes.rst file is allowed but not mandatory: we don't change it on snapshots, and while the release check itself checks separately for "is this a snapshot?" and "is this a stable one?" and thus may misclassify, it only outputs whether it's a release or not.

The cron job (and a few release jobs) checks out the release-notes.rst file as of the trigger commit rather than the target commit. Assuming the release notes page stops changing, you could argue this is unnecessary, but it's also not harmful and I think useful to keep during the transition, where we might still target commits older than this change in future releases.

So yes, they should probably be update to match, but I don't think that should be done as part of this PR and I don't think it should be rushed either.

File: release/RELEASE.md (hunk 1 above)
Comment (garyverhaegen-da) [reply release/RELEASE.md:61]: I followed the instructions here so not sure. However, the node install is ~60MB and the yarn install is ~1.5MB, so my first guess would be no.

**Comment by garyverhaegen-da** [comment]
@cocreature I've added the redirect. Two notes:

1. I have double-checked that, as of right now, the release notes page is the only one that gets clobbered by the `cp -f` command. I believe this is the right behaviour, but I can see how this could seem dangerous, so I'll let you make the call here. However, for this specific case, removing the `release-notes.rst` file entirely seems like a bad idea re: the rest of the release process.
2. I personally prefer the extra click to the unexpected break in navigation, but it's your call.

**Comment by cocreature** [comment]
Alright, you convinced me. Let’s leave out the redirect for now. At least until we’ve cleared up all references to our release notes in docs cron and so on so we could get rid of it completely and avoid the `-f`.

**Comment by garyverhaegen-da** [comment]
> Alright, you convinced me. Let’s leave out the redirect for now. At least until we’ve cleared up all references to our release notes in docs cron and so on so we could get rid of it completely and avoid the `-f`.

To clarify, I think the `-f` is the right behaviour for the redirects build; I'm more annoyed by the unexpected change of website when clicking on what looks like a navigation menu.

**Pull request merged**